app/Providers/ImageHelperServiceProvider.php

```diff
@@ -24,9 +24,9 @@ public function boot()
      */
     public function register()
     {
-        $this->app->bind('helper',function()
+        $this->app->bind('imagehelper',function()
         {
-            return new Helper;
+            return new ImageHelper;
```

Comment:
IMO returning just an empty class might not need to have a service provider and binding